moveit_core/python/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(MOVEIT_LIB_NAME moveit_python)
```

Comment:
In MTC I have used `moveit_python_tools` as lib name.

Reply:
yea I like that better as well, I'll change it

Reply:
I think you also want to avoid moveit_python as a name, because there is also a (released) ROS package called [moveit_python](https://github.com/mikeferguson/moveit_python) and the imports might get clouded for people who have that installed